src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  height: 406px;
+  width: 198px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+  }
+
+  &__code {
+    margin: 4px 16px 0;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Repeat the previous comment about font-family. Use a variable for the font-family to maintain consistency and reusability.